### PR TITLE
bug bash for sensitivity_pruner

### PR DESCRIPTION
**Pull request by zheng-ningxin**
Bug bash for the sensitivity_pruner

### CODE REVIEW

**Review by chicm-ms — APPROVED** [review]

File: src/sdk/pynni/nni/compression/torch/pruning/sensitivity_pruner.py [hunk 1]
```diff
@@ -383,14 +394,20 @@ def compress(self, eval_args=None, eval_kwargs=None,
...
                 # sensitivity analysis
+
                 self.analyzer.load_state_dict(self.model.state_dict())
                 self.sensitivities = self.analyzer.analysis(
                     val_args=eval_args, val_kwargs=eval_kwargs)
 
         _logger.info('After Pruning: %.2f weights remains', cur_ratio)
+        self.modules_wrapper = modules_wrapper_final
+        # TODO double check if return a normal model or
+        # a wrapped model
```
Comment (QuanluZhang) [inline src/sdk/pynni/nni/compression/torch/pruning/sensitivity_pruner.py:409]: could you check the model and remove this TODO?

File: src/sdk/pynni/nni/compression/torch/utils/sensitivity_analysis.py [hunk 2]
```diff
@@ -163,28 +163,30 @@ def analysis(self, val_args=None, val_kwargs=None, specified_layers=None):
         if val_kwargs is None:
             val_kwargs = {}
         # Get the original validation metric(accuracy/loss) before pruning
-        if self.ori_metric is None:
-            self.ori_metric = self.val_func(*val_args, **val_kwargs)
+        # Get the accuracy baseline before staring the analysis.
```
Comment (QuanluZhang) [inline src/sdk/pynni/nni/compression/torch/utils/sensitivity_analysis.py:166]: staring -> starting

Comment (zheng-ningxin) [reply src/sdk/pynni/nni/compression/torch/utils/sensitivity_analysis.py]: Fixed, thanks~

**Review by QuanluZhang — APPROVED** [review]

File: src/sdk/pynni/nni/compression/torch/pruning/sensitivity_pruner.py (hunk 1 above)
Comment (zheng-ningxin) [reply src/sdk/pynni/nni/compression/torch/pruning/sensitivity_pruner.py]: Checked and updated.

**Pull request merged**